### PR TITLE
Ship VPN exclude app prompt

### PR DESCRIPTION
**Pull request by karlenDimla**
Task/Issue URL: https://app.asana.com/0/1142021229838617/1208316612251365/f

### Description
Enable the exclude app prompt shown everytime the VPN is attempted to be disabled by the user.

### Steps to test this PR
https://app.asana.com/0/1142021229838617/1208233455261636/f

### CODE REVIEW

**Comment by karlenDimla** [comment]
* **#5028** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5028?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karlenDimla and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: network-protection/network-protection-impl/src/test/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementViewModelTest.kt [hunk 1]
```diff
@@ -197,6 +197,7 @@ class NetworkProtectionManagementViewModelTest {
 
     @Test
     fun whenOnNetpToggleClickedToDisabledThenUnregisterFeature() = runTest {
+        vpnRemoteFeatures.showExcludeAppPrompt().setEnabled(Toggle.State(enable = false))
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/test/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementViewModelTest.kt:200]: @aitorvs is this not correct way of setting the feature state in test? I just see that it is being flagged in lint

Comment (aitorvs) [reply network-protection/network-protection-impl/src/test/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementViewModelTest.kt:200]: It is, just add a `Suppress` annotation

**Review by aitorvs — APPROVED** [review]
LGTM

**Pull request merged**